lib/biz/timeline/backward.rb

```diff
@@ -11,9 +11,17 @@ def backward
       private
 
       def occurred?(period, time)
+        period.end_time <= time
+      end
+
+      def elapsed?(period, time)
```

Comment:
Elapsed definition
> (of time) pass or go by

When i think of elapsed i think of it as a synonym for "occured". Would `started?` be more accurate here?

Reply:
Actually, the final form rendered the need for the `elapsed` check useless (it always returned `false`). I've removed it.